packages/rpc-core/src/rpc-methods/getBlockTime.ts

```diff
@@ -0,0 +1,14 @@
+import { U64UnsafeBeyond2Pow53Minus1 } from './common';
+
+/** Estimated production time, as Unix timestamp (seconds since the Unix epoch) */
+type GetBlockTimeApiResponse = U64UnsafeBeyond2Pow53Minus1;
+
+export interface GetBlockTimeApi {
+    /**
+     * Returns the estimated production time of a block.
+     */
+    getBlockTime(
+        /** block number, identified by Slot */
+        blockNumber: U64UnsafeBeyond2Pow53Minus1
```

Comment:
Shall we just use the `Slot` type?
```suggestion
        blockNumber: Slot,
```